include/slate/func.hh

```diff
@@ -41,6 +41,29 @@ inline std::function<int64_t(int64_t)> uniform_blocksize(int64_t n, int64_t nb)
     return [n, nb](int64_t j) { return (j + 1)*nb > n ? n%nb : nb; };
 }
 
+//------------------------------------------------------------------------------
+/// Computes the largest block given a blocksize function
```

Comment:
Sentences should end with period, particularly in Doxygen since it uses the first sentence as the brief description.